src/components/DigitalCredentials/CredentialsTable.vue

```diff
@@ -10,72 +10,68 @@
         fixed
         disable-sort
         hide-default-footer
-        :items="credentialRecords"
-        :headers="!!credentialRecords ? credentialsTableHeaders : []"
+        :items="issuedCredentials"
+        :headers="!!issuedCredentials ? credentialsTableHeaders : []"
         no-data-text="All digital credentials associated with this business will appear here."
       >
+        <template v-slot:item.credentialType="{ item }">
+          {{formatCredentialType(item.credentialType)}}
+        </template>
+        <template v-slot:item.isIssued="{ item }">
+          {{formatStatus(item.isIssued)}}
+        </template>
+        <template v-slot:item.dateOfIssue="{ item }">
+          {{apiToPacificDate(item.dateOfIssue) || '-'}}
+        </template>
       </v-data-table>
     </v-card>
   </div>
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator'
-import { TableHeaderIF } from '@/interfaces'
+import { Component, Mixins, Prop } from 'vue-property-decorator'
+import { DigitalCredentialsIF, TableHeaderIF } from '@/interfaces'
+import { DigitalCredentialTypes } from '@/enums'
+import { DateMixin } from '@/mixins'
 
 @Component({})
-export default class CredentialsTable extends Vue {
+export default class CredentialsTable extends Mixins(DateMixin) {
+  @Prop({ default: [] })
+  readonly issuedCredentials: Array<DigitalCredentialsIF>
```

Comment:
Please make this 1 line and add the Typescript null assertion character.

Reply:
Updated!